easybuild/easyblocks/t/tensorflow.py

```diff
@@ -73,6 +76,130 @@
 """
 
 
+def split_tf_libs_txt(valid_libs_txt):
+    """Split the VALID_LIBS entry from the TF file into single names"""
+    entries = valid_libs_txt.split(',')
+    # Remove double quotes and whitespace
+    result = [entry.strip().strip('"') for entry in entries]
+    # Remove potentially trailing empty element due to trailing comma in the txt
+    if not result[-1]:
+        result.pop()
+    return result
+
+
+def get_system_libs_from_tf(source_dir):
+    """Return the valid values for TF_SYSTEM_LIBS from the TensorFlow source directory"""
+    syslibs_path = os.path.join(source_dir, 'third_party', 'systemlibs', 'syslibs_configure.bzl')
+    result = []
+    if os.path.exists(syslibs_path):
+        txt = read_file(syslibs_path)
+        m = re.search(r'VALID_LIBS = \[(.*?)\]', txt, re.DOTALL)
+        if not m:
+            raise EasyBuildError('VALID_LIBS definition not found in %s', syslibs_path)
```

Comment:
@Flamefire This is too strict, it breaks the installation of `TensorFlow-1.10.0-fosscuda-2018b-Python-2.7.15.eb` for example.

It seems OK for 1.13.1 and newer, so I would only error out for sufficiently recent versions?

Also, we shouldn't use a single-letter variable outside of a list comprehension, so please fix that too while you're at it (`valid_libs = re.search(...)`)

Reply:
Found that 1.10.0 (where this was introduced) is missing spaces. Adapted the regex.

It is actually only done for "sufficiently recent versions" where I defined this as "has the `syslibs_configure.bzl` file", which should be ok enough.

Fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2117/commits/969fcf245727d88b7dd7cb1cfd7154e380e98f22